...s.............................................................F
=================================== FAILURES ===================================
__________________ TestFastAPIEndpoints.test_health_endpoint ___________________

self = <tests.test_phase1.TestFastAPIEndpoints object at 0x1048c6f80>

    def test_health_endpoint(self):
        """Test health endpoint."""
>       from main import app

tests/test_phase1.py:302: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
main.py:21: in <module>
    from pipelines.evidently_monitoring import evidently_monitoring_step
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    """
    Evidently Monitoring Pipeline Step
    ---------------------------------
    Generates data/model drift reports using Evidently AI.
    """
    import pandas as pd
>   from evidently.report import Report
E   ModuleNotFoundError: No module named 'evidently.report'

pipelines/evidently_monitoring.py:7: ModuleNotFoundError
=========================== short test summary info ============================
FAILED tests/test_phase1.py::TestFastAPIEndpoints::test_health_endpoint - Mod...
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
1 failed, 64 passed, 1 skipped, 10 warnings in 33.89s
